app/scripts/models/email-resend.js

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// This model limits the number of emails a component can send
+
+'use strict';
+
+define([
+  'backbone',
+  'lib/constants'
+], function (Backbone, Constants) {
+
+  function shouldResend (tries) {
+    return tries === 1 || tries === 4;
```

Comment:
Instead of hard coding 4 here, maybe pass in `maxTries` and use that instead. Update `EMAIL_RESEND_MAX_TRIES` to be 4. That seems clearer than setting `EMAIL_RESEND_MAX_TRIES` to 3, and still sending an email on the 4th attempt.